### PR TITLE
Bid adapter PGAMSSP: new adapter

### PR DESCRIPTION
**Pull request by PGAMSSP**
doc - https://github.com/prebid/prebid.github.io/pull/4810

### CODE REVIEW

File: modules/pgamsspBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
+import { isFn, deepAccess, logMessage, logError } from '../src/utils.js';
+import { convertOrtbRequestToProprietaryNative } from '../src/native.js';
+
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import { config } from '../src/config.js';
+
+const BIDDER_CODE = 'pgamssp';
```
Comment (ncolletti) [inline modules/pgamsspBidAdapter.js:8]: Your bidder docs mention being tcfeu supported, we will need your GVL ID added to the adapter

File: modules/pgamsspBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+        response.push(resItem);
+      }
+    }
+    return response;
+  },
+
+  getUserSyncs: (syncOptions, serverResponses, gdprConsent, uspConsent) => {
+    let syncType = syncOptions.iframeEnabled ? 'iframe' : 'image';
+    let syncUrl = SYNC_URL + `/${syncType}?pbjs=1`;
```
Comment (ncolletti) [inline modules/pgamsspBidAdapter.js:190]: This sync url is not working

File: modules/pgamsspBidAdapter.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                    mediaTypes: {
+                        banner: {
+                            sizes: [ [300, 250], [320, 50] ],
+                        }
+                    },
+                    bids: [
+                        {
+                            bidder: 'pgamssp',
+                            params: {
+                                placementId: 'testBanner',
```
Comment (ncolletti) [inline modules/pgamsspBidAdapter.md:29]: We'll need to see each example in your adapter's md file to return a valid bid response

File: modules/pgamsspBidAdapter.js (hunk 2 above)
Comment (PGAMSSP) [reply modules/pgamsspBidAdapter.js:190]: fixed

File: modules/pgamsspBidAdapter.js (hunk 1 above)
Comment (PGAMSSP) [reply modules/pgamsspBidAdapter.js:8]: we updated doc

**Review by ncolletti — APPROVED** [review]
looks good to me

**Pull request merged**